### DISCUSSION TITLE
Re-Enabling Legacy Components

### DISCUSSION THREAD

**Discussion by ghost** [farag2/Sophia-Script-for-Windows#189] [Q&A] [answered]
I couldn't find an option to turn it back on, only to disable it on windows features pop-out from the script.

**Comment by farag2** [comment, marked as answer]
What about

https://github.com/farag2/Windows-10-Sophia-Script/blob/73de164d1936868daeb5bbe60e19ff6eadfeb218/Sophia/PowerShell%205.1/Sophia.ps1#L676

Invoke the function via this intruction
https://github.com/farag2/Windows-10-Sophia-Script#how-to-run-the-specific-functions

**Reply by ghost** [reply]
That is what confused me, when I tried to invoke that function with enable it only showed the following options. So I'm wondering if there's any way to enable it trough the functions
![WindowsFeatures-Enable](https://user-images.githubusercontent.com/53121679/121176361-83cb7080-c832-11eb-8521-b2756c434432.png)

**Reply by farag2** [reply]
It shows only disabled features. Logic!

**Reply by ghost** [reply]
ohhh ok, since i`ve been having problems enabling DirectPlay I assumed it was turned off by the script but that was never the issue to begin with, thanks for clarifying it for me.

**Reply by farag2** [reply]
You're welcome!
